main/nav_main_offset_cloud_tasks.py

```
@@ -0,0 +1,132 @@
+################################################################################
```

Ruff (0.14.5)

The file is executable but no shebang is present

(EXE002)